### PR TITLE
SC-1073 play upgradation changes

### PR DESCRIPTION
**Pull request by Hari-stackroute**
pom changes -> akka lib upgradation
controller changes -> used CompletionStage instead of promise
test cases changes.

### CODE REVIEW

File: service/app/modules/StartModule.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package modules;
+
+import com.google.inject.AbstractModule;
+
+public class StartModule extends AbstractModule {
+
+  @Override
+  protected void configure() {
+    System.out.println("StartModule:configure: Start");
```
Comment (manzarul) [inline service/app/modules/StartModule.java]: remove system.out .print used logger

File: service/conf/application.conf [hunk 2]
```diff
@@ -59,8 +59,11 @@ play.http {
   # Default to Routes in the root package (aka "apps" folder) (and conf/routes)
   #router = my.application.Router
   #TO allow more data in request body
+  errorHandler = modules.ErrorHandler
+  actionCreator = modules.OnRequestHandler
```
Comment (manzarul) [inline service/conf/application.conf]: #TO allow more data in request body need to come below "actionCreator = modules.OnRequestHandler"

File: service/app/controllers/skills/SkillController.java [hunk 3]
```diff
@@ -1,67 +0,0 @@
-package controllers.skills;
```
[the remark this answers is not in the document]
Comment (Hari-stackroute) [reply service/app/controllers/skills/SkillController.java]: Same is there in UserSkillController and no where SkillController is called. so removed.

File: service/app/controllers/usermanagement/UserController.java [hunk 4]
```diff
@@ -43,36 +45,38 @@
...
   }
 
-  public Promise<Result> getUserById(String userId) {
+  public CompletionStage<Result> getUserById(String userId, Http.Request httpRequest) {
     return handleGetUserProfile(
-        ActorOperations.GET_USER_PROFILE.getValue(), ProjectUtil.getLmsUserId(userId));
+        ActorOperations.GET_USER_PROFILE.getValue(), ProjectUtil.getLmsUserId(userId), httpRequest);
   }
 
   public Result getUserByIdMock(String userId) {
```
[the remark this answers is not in the document]
Comment (Hari-stackroute) [reply service/app/controllers/usermanagement/UserController.java]: Done

File: service/app/modules/StartModule.java (hunk 1 above)
Comment (Hari-stackroute) [reply service/app/modules/StartModule.java]: Done

File: service/conf/application.conf (hunk 2 above)
Comment (Hari-stackroute) [reply service/conf/application.conf]: Done

**Pull request merged**